spec/unit/services_inspector_spec.rb

```diff
@@ -45,10 +45,17 @@
 EOF
     }
 
+    let(:chkconfig_redhat_output) {
+      <<-EOF
+crond 0:off 1:off 2:on 3:on 4:on 5:on 6:off
+dnsmasq 0:off 1:off 2:off 3:off 4:off 5:off 6:off
+EOF
+    }
```

Comment:
This is only used once. The code would be more easy to read if this would be defined at the place where it's used. Then we also don't need a `let`.